rollup.config.js

```diff
@@ -0,0 +1,85 @@
+import babel from 'rollup-plugin-babel'
+import commonjs from '@rollup/plugin-commonjs'
+import replace from '@rollup/plugin-replace'
+import resolve from '@rollup/plugin-node-resolve'
+import { terser } from 'rollup-plugin-terser'
+import pkg from './package.json'
+
+const { peerDependencies } = pkg
+const external = Object.keys(peerDependencies)
+
+const namedExports = {
+  'node_modules/react-is/index.js': ['isForwardRef'],
+  'node_modules/react-tiny-virtual-list/node_modules/prop-types/index.js': [
+    'array',
+    'arrayOf',
+    'func',
+    'number',
+    'object',
+    'oneOf',
+    'oneOfType',
+    'string'
+  ]
+}
+
+export default [
+  // UMD Development
```

Comment:
How does rollup know that this is development and the 2nd item in the exported array is production?

Reply:
Rollup doesn't know about it. The first I specified is development because I don't use something optimization. And in the 2nd item, I use `terser()` to minify the generated.